src/Illuminate/View/Engines/CompilerEngine.php

```diff
@@ -56,8 +56,18 @@ public function get($path, array $data = [])
         // Once we have the path to the compiled file, we will evaluate the paths with
         // typical PHP just like any other templates. We also keep a stack of views
         // which have been rendered for right exception messages to be generated.
-        $results = $this->evaluatePath($compiled, $data);
-
+        $results  = $this->evaluatePath($compiled, $data);
+        if (env("THEME_HINT_ENABLED")) {
```

Comment:
I don`t know a correct file to define configuration for Laravel View.
Do I have to add it to config\view.php it seems store configuration for Laravel View ?
But I assume .env file seems to be simpler than config\view.php.
You don`t get error altought configuration is not exists.